index.html

```diff
@@ -35,5 +35,118 @@ <h1>Howdy! I am Flover. <span> I am a software developer</span></h1>
         <a href="#middle" id="down-button" class="fa-solid fa-chevron-down"></a>
       </header>
     </section>
+
+    <main id="middle" class="font-style">
+      <h3>My Recent Works</h3>
+      <div class="container">
+        <article class="grid-item">
+          <img src="https://placehold.jp/350x250.png" alt="Placeholder Image">
+          <h5>Multi-Post Stories <span>Gain+Glory</span></h5>
+          <ul class="list-format list-container">
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
```

Comment:
- [ ] This project cards container is expected to use `display: grid` to align its contents which are the various cards. Kindly fix this